### PR TITLE
Add support for Asus Zenfone 2 Laser 720p (Z00L)

### PR DESCRIPTION
**Pull request by antony-jr**
This PR adds support to boot lk2nd in Asus Zenfone 2 Laser 720p (Z00L).

### CODE REVIEW

File: dts/msm8916-asus-z00l.dts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+        qcom,msm-id =   <206 0>,
+			<248 0>,
+			<249 0>,
+			<250 0>;
```
Comment (Minecrell) [inline dts/msm8916-asus-z00l.dts:11]:
```
qcom,msm-id = <206 0>;
```

should be enough

File: dts/msm8916-asus-z00l.dts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	qcom,board-id = <21 0>;
+	compatible = "qcom,msm8916-mtp", "qcom,msm8916";
+	
+	asus-z00l {
```
Comment (Minecrell) [inline dts/msm8916-asus-z00l.dts:15]: Your original changes in the first commit were fine, this special syntax is only needed when multiple devices use one board-id.

**Review by Minecrell — CHANGES_REQUESTED** [review]

File: dts/msm8916-asus-z00l.dts (hunk 2 above)
Comment (antony-jr) [reply dts/msm8916-asus-z00l.dts:15]: See https://github.com/msm8916-mainline/lk2nd/blob/3f6837d645d36e27a4a0a4298a91e52ce6c0febe/dts/msm8939-asus-z00t.dts#L10 , Asus Z00T is very similar to Z00L but uses a different SOC but the board id seems to be similar.

File: dts/msm8916-asus-z00l.dts (hunk 1 above)
Comment (antony-jr) [reply dts/msm8916-asus-z00l.dts:11]: Alright.

File: dts/msm8916-asus-z00l.dts (hunk 2 above)
Comment (Minecrell) [reply dts/msm8916-asus-z00l.dts:15]: https://github.com/msm8916-mainline/lk2nd/commit/1f5a335422866c8f3e952cfce8ebc8083a85a531

**Comment by Minecrell** [comment]
I mentioned it on Matrix already but FWIW:

Feel free to include gpio-keys and wifi (pronto) already here. I would squash that into the initial commit later anyway. :)

**Comment by antony-jr** [comment]
> Feel free to include gpio-keys and wifi (pronto) already here.

I think you confused with the other PR, forgive me for using similar names for the PR. This is lk2nd.

**Comment by Minecrell** [comment]
> > Feel free to include gpio-keys and wifi (pronto) already here.
> 
> I think you confused with the other PR, forgive me for using similar names for the PR. This is lk2nd.

Yeah I already deleted it here and added it to the other one xD

**Comment by Minecrell** [comment]
Removed some whitespace after `qcom,msm-id`, squashed the commits and pushed it. Thanks!

**Pull request closed without merging**